hydrolib/tools/ext_old_to_new/converters.py

```diff
@@ -119,7 +133,52 @@ class BoundaryConditionConverter(BaseConverter):
     def __init__(self):
         super().__init__()
 
-    def convert(self, forcing: ExtOldForcing) -> Boundary:
+    @staticmethod
+    def parse_tim_model(tim_files: List[TimModel], forcing: ExtOldForcing) -> TimModel:
+        """Parse the boundary condition related time series from the tim files.
+
+        Args:
+            tim_files (List[TimModel]): List of TIM models.
+            forcing (ExtOldForcing):
+
+        Returns:
+            TimModel: A TimModel object containing the time series data from the TIM files.
+        """
+        time_files_exist = all([tim_file.exists() for tim_file in tim_files])
+        if not time_files_exist:
+            raise ValueError(
+                f"TIM files '{tim_files}' not found for QUANTITY={forcing.quantity}"
+            )
+
+        tim_models = [
+            TimModel(file, quantities_names=[file.stem]) for file in tim_files
+        ]
+        # merge all the tim files into one tim model
+        for tim_model in tim_models[1:]:
+            data = tim_model.as_dict()
+            if len(data.keys()) != 1:
+                raise ValueError(
+                    f"Number of columns in the TIM file '{tim_model.filepath}' should be 1 column."
```

Comment:
Actually 2 (the user sees the time column and then the data column(s)).

Reply:
updated the error message.